### PR TITLE
added simpsons rule in c++ and java

### LINKED ISSUE #1619: Simpsons rule implementation in c++ and java
I want to work on this issue for GSSOC 2020

### PR DESCRIPTION
**Pull request by yuktirapartiwar**
Fixes #1619 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Simpsons rule in C++ and java
- 

#### Languages Used:
- C++
- java

#### Files Added:
- Simpsons_Rule.cpp
- Simpsons_Rule.java

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by MastersAbh** [comment]
Please mention the issue # you are fixing in the changelist

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Default ignored files
+/workspace.xml
```
Comment (MastersAbh) [inline .idea/.gitignore]: Remove this file from commit

File: .idea/Algo_Ds_Notes.iml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (MastersAbh) [inline .idea/Algo_Ds_Notes.iml]: Remove this file from commit

File: .idea/misc.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" project-jdk-name="11.0.5" project-jdk-type="JavaSDK" />
+</project>
```
Comment (MastersAbh) [inline .idea/misc.xml]: Remove this file from commit

File: .idea/modules.xml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/Algo_Ds_Notes.iml" filepath="$PROJECT_DIR$/.idea/Algo_Ds_Notes.iml" />
+    </modules>
+  </component>
+</project>
```
Comment (MastersAbh) [inline .idea/modules.xml]: Remove this file from commit

File: .idea/vcs.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
```
Comment (MastersAbh) [inline .idea/vcs.xml]: Remove this file from commit

**Comment by yuktirapartiwar** [comment]
@MastersAbh Resolved

**Pull request merged**